committeeoversightapp/models.py

```diff
@@ -85,8 +85,8 @@ def url(self):
         return '/committee-' + self.id.split('ocd-organization/').pop()
 
     @property
-    def parent_url(self):
-        return '/committee-' + self.parent.id.split('ocd-organization/').pop()
+    def parent_proxy(self):
```

Comment:
Ooh, this is venturing very close to the behavior we implemented in `django-councilmatic` with [`django-proxy-overrides`](https://github.com/datamade/django-proxy-overrides). More on that here: https://github.com/datamade/django-councilmatic/tree/2.5#proxying

Meanwhile, this is a smart solution for limited needs! Just be aware it will fire at least two SQL queries – one to get the parent, and the other to get the `CommitteeOrganization` object – so it won't scale as well.

Reply:
Ah very cool to see, thanks for the links! I'm going to leave this for now but if we find that we're using these parent properties more, or if the current ones are slowing down the site, I'll plan to switch it.